### PR TITLE
Add "check_label_and_comment" workflow

### PR DESCRIPTION
**Pull request by M0rgan01**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 9.0.x
| Description?      |  This change allows you to automatically add a comment explaining the contributions to the autoupgrade repository to the PR if the conditions are met (the Need autoupgrade PR label as well as 2 approves)
| Type?             | improvement
| Category?         | PM
| BC breaks?        | no
| Deprecations?   |   no
| How to test?      | We need to merge the workflow and then test it on another PR -> https://github.com/M0rgan01/PrestaShop/pull/20#issuecomment-2506010448
| UI Tests          | -
| Fixed issue or discussion?     | -
| Related PRs       | -
| Sponsor company   | -

### CODE REVIEW

**Review by tblivet — DISMISSED** [review]

**Comment by ps-jarvis** [comment]
Hi, thanks for this contribution!

I found some issues with the Pull Request description:

* The `deprecations` should be `yes` or `no`. ([Read explanation](https://devdocs.prestashop-project.org/9/contribute/contribution-guidelines/pull-requests/#deprecations))

Would you mind having a look at it? This will help us understand how interesting your contribution is, thank you very much!

(Note: this is an automated message, but answering it will reach a real human)

**Review by tblivet — DISMISSED** [review]

**Review by ga-devfront — DISMISSED** [review]

**Review by tblivet — DISMISSED** [review]

**Review by ga-devfront — DISMISSED** [review]

**Review by tblivet — DISMISSED** [review]

**Review by M0rgan01 — COMMENTED** [review]
Hello @$PR_CREATOR,<br/>
            Please note that the “Needs autoupgrade PR” label has been automatically added to your Pull Request. This means that the changes made in your PR must also be reported in the Autoupgrade module, responsible for running the update process of PrestaShop stores.<br/>
            When there's a modification to the \`db-structure.sql\` file or if a Doctrine entity has been modified, added, or deleted, it impacts the database structure. Modifying the list of hooks and feature flags has consequences on the update process as well. These changes can affect how the application interacts with the database and how update procedures are handled. It's important to consider these aspects when contributing to ensure a smooth update experience for users.<br/>
            That's why this contribution should be backported, with another corresponding PR, on the Autoupgrade module's GitHub repository which is available here: [Autoupgrade Repository](https://github.com/PrestaShop/autoupgrade)
            Contribution to the Autoupgrade module documentation: [Documentation Link](https://devdocs.prestashop-project.org/9/contribute/contribution-guidelines/upgrade-module/)<br/>
            Don't forget to add a comment in this current thread with the link to this PR once it has been created.
            If you are sure that your Pull Request doesn’t require additional actions, ping committers and it will be checked.<br/>
            Thank you.

File: .github/workflows/verify_autoupgrade_label_and_notify.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            When there's a modification to the \`db-structure.sql\` file or if a Doctrine entity has been modified, added, or deleted, it impacts the database structure. Modifying the list of hooks and feature flags has consequences on the update process as well. These changes can affect how the application interacts with the database and how update procedures are handled. It's important to consider these aspects when contributing to ensure a smooth update experience for users.<br/>
+            That's why this contribution should be backported, with another corresponding PR, on the Autoupgrade module's GitHub repository which is available here: [Autoupgrade Repository](https://github.com/PrestaShop/autoupgrade)
+            Contribution to the Autoupgrade module documentation: [Documentation Link](https://devdocs.prestashop-project.org/9/contribute/contribution-guidelines/upgrade-module/)<br/>
+            Don't forget to add a comment in this current thread with the link to this PR once it has been created.
```
Comment (Touxten) [inline .github/workflows/verify_autoupgrade_label_and_notify.yml]:
```suggestion
            Don't forget to add a comment in this current thread with the link to this PR once it has been created.<br/>
```

File: .github/workflows/verify_autoupgrade_label_and_notify.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            If you are sure that your Pull Request doesn’t require additional actions, ping committers and it will be checked.<br/>
```
Comment (Touxten) [inline .github/workflows/verify_autoupgrade_label_and_notify.yml]:
```suggestion
```

not possible :'(

**Review by Touxten — COMMENTED** [review]
Small feedback regards

File: .github/workflows/verify_autoupgrade_label_and_notify.yml (hunk 2 above)
Comment (M0rgan01) [reply .github/workflows/verify_autoupgrade_label_and_notify.yml]: how is that not possible?

Comment (Touxten) [reply .github/workflows/verify_autoupgrade_label_and_notify.yml]: Hello, 

https://github.com/PrestaShop/docs/pull/1889

Comment (Touxten) [reply .github/workflows/verify_autoupgrade_label_and_notify.yml]: A link to : https://www.prestashop-project.org/project-organization/people-and-roles/#committers

**Review by Touxten — APPROVED** [review]

File: .github/workflows/verify_autoupgrade_label_and_notify.yml (hunk 2 above)
Comment (M0rgan01) [reply .github/workflows/verify_autoupgrade_label_and_notify.yml]: Good idea

**Review by Quetzacoalt91 — APPROVED** [review]

**Review by ga-devfront — APPROVED** [review]

**Pull request merged**